sync/src/test/java/tech/pegasys/artemis/sync/PeerSyncTest.java

```diff
@@ -193,6 +193,11 @@ void sync_stoppedBeforeBlockImport() {
     assertThat(syncFuture).isCompleted();
     PeerSyncResult result = syncFuture.join();
     assertThat(result).isEqualByComparingTo(PeerSyncResult.CANCELLED);
+
+    // check startingSlot
+    UnsignedLong startingSlot = peerSync.getStartingSlot();
+    assertThat(startingSlot).isGreaterThan(UnsignedLong.valueOf(0));
```

Comment:
Nit: UnsignedLong.ZERO

Reply:
Shouldn't we know the exact starting slot?  We setup the remote peer state and control when it responds with data, so it should be completely deterministic from what I can see (but there's a good chance I'm missing something)